janusgraph-es/pom.xml

```diff
@@ -21,6 +21,7 @@
         <skip.es.test>${skipTests}</skip.es.test>
         <es.docker.image>docker.elastic.co/elasticsearch/elasticsearch</es.docker.image>
         <elasticsearch.docker.test.version>${elasticsearch.test.version}</elasticsearch.docker.test.version>
+        <powermock.version>1.6.1</powermock.version>
```

Comment:
Normally for 1.8.5 Mockito you should have powermock between 1.3.9 and 1.4.9.
Either way powermock 1.6.1 is not reference as stable.
(https://github.com/powermock/powermock/wiki/Mockito)

Reply:
I know, I remember I was having problems at some point with my tests and Java8.  Turned out, it was a problem with Javassist and Java8. And to solve it I needed to bump Powermock version. And going to the latest 1.6.x was causing some other horrible issues. So I have kept 1.6.1 (worked) hoping that we can revisit it once Mockito version is updated. I am going to retest with 1.4.9 now to see if I am still hitting that bug.

Reply:
Yes, I hit this problem: https://github.com/powermock/powermock/issues/551. It seems until there is a new release of JUnit, it can be worked around either by downgrading JUnit to 4.11 or by bumping up PowerMock to 1.6.1 I have tried the latter approach and it worked. Which one would you prefer?